systems/subscriber/registry/pb/subscriber.proto

```diff
@@ -87,6 +87,18 @@ message GetSubscriberResponse{
 ukama.common.v1.Subscriber subscriber = 1;
 }
 
+// GetSubscriberByEmailRequest message is used to get a subscriber by subscriber email
+message GetSubscriberByEmailRequest{
+// email field is used to get subscriber by email
+string email = 1 [(validator.field) = { human_error:"must be an email format" regex: "^$|^[a-z0-9._%+\\-]+@[a-z0-9.\\-]+\\.[a-z]{2,4}$"}];
```

Comment:
Need to use validation from common.

Reply:
Validator is already coming from common.

Reply:
Then we don't need to have extra validation here it should just be string